zpl/label.py

```diff
@@ -179,6 +180,25 @@ def write_field_number(self, number, name=None, char_height=None, char_width=Non
                 "characters and spaces"
             self.code += '"%s"' % name
 
+    def write_bar_code(self, height, barcode_type, oritentation='N', check_digit='N',
```

Comment:
I would personally prefer `write_barcode`, also for all variables containing `bar_code` (to be replaced with `barcode`)

Reply:
Check the latest commit, I believe I've refactored everything. 

Thanks for the quick response!

Reply:
The example code in the README.rst file needs to be refactored too :)

Reply:
Nevermind